examples/instancing/main.rs

```diff
@@ -146,29 +150,37 @@ impl<R: gfx::Resources> gfx_app::Application<R> for App<R> {
                 locals: factory
                     .create_buffer_immutable(&[locals], gfx::buffer::Role::Constant, gfx::Bind::empty())
                     .unwrap(),
-                out: window_targets.color,
+                out: window_targets.views[0].0.clone(),
             },
             slice: slice,
             upload: upload,
             uploading: true,
+            views: window_targets.views,
         }
     }
 
-    fn render<C: gfx::CommandBuffer<R>>(&mut self, encoder: &mut gfx::Encoder<R, C>) {
+    fn render<Gp>(&mut self, (frame, semaphore): (gfx::Frame, &gfx::handle::Semaphore<B::Resources>),
+                  pool: &mut Gp, queue: &mut gfx::queue::GraphicsQueueMut<B>)
+        where Gp: gfx::GraphicsCommandPool<B>
+    {
+        let mut encoder = pool.acquire_graphics_encoder();
```

Comment:
Possible, a valid request as a single Encoder the common case for our examples. I'll look into it after updated the other examples.